docs/update.md

```diff
@@ -31,9 +31,10 @@ php flarum cache:clear
 ### Updating from Beta (<=0.1.0) to Stable v1 (^1.0.0)
 
 1. Do steps 1-5 above.
+3. Change the version strings of all bundled extensions (`flarum/tags`, `flarum/mentions`, `flarum/likes`, etc) in `composer.json` from `^0.1.0` to `*`.
```

Comment:
I know in the rendering it doesn't matter, but is it intentional to have 1-3-2 instead of 1-2-3? :innocent: